CMakeLists.txt

```diff
@@ -303,6 +303,17 @@ if(CONFIG_OVERRIDE_FRAME_POINTER_DEFAULT)
   endif()
 endif()
 
+if($ENV{ZEPHYR_BUILD_COLOR_OUTPUT})
```

Comment:
What's the reason for controlling this through environment variables ?

Reply:
With `=always` the ANSI color tags will always be printed. It will probably make any web apps printing the terminal output un-readable. Therefore this should be enabled manually.

Reply:
yes, but why should it be enabled through an undocumented environment setting ?